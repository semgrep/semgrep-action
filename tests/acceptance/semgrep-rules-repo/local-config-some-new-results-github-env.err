| versions          - semgrep x.y.z on Python x.y.z
| environment       - running in environment github-actions, triggering event is 'unknown'
| manage            - not logged in
=== setting up agent configuration
| using semgrep rules from python/django/security/audit/xss/template-translate-as-no-escape.yaml
| using default path ignore rules of common test and dependency directories
| reporting findings introduced by these commits:
|   * aaaaaaa XSS via translate filters in Django Templates (#1071)
| looking at 6 changed paths
| found 6 files in the paths to be scanned
=== looking for current issues in 6 files
| 2 current issues found
| No issues muted with nosemgrep comment
=== not looking at pre-existing issues since all files with current issues are newly created
=== github annotation output
=== exiting with failing status
