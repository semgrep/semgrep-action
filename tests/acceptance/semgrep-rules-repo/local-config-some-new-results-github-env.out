::notice file=python/django/security/audit/xss/template-translate-as-no-escape.html,line=8,endLine=9,title=semgrep rule:python.django.security.audit.xss.template-translate-as-no-escape::Translated strings will not be escaped when rendered in a template. This leads to a vulnerability where translators could include malicious script tags in their translations. Consider using `force_escape` to explicitly escape a transalted text.
::notice file=python/django/security/audit/xss/template-translate-as-no-escape.html,line=2,endLine=4,title=semgrep rule:python.django.security.audit.xss.template-translate-as-no-escape::Translated strings will not be escaped when rendered in a template. This leads to a vulnerability where translators could include malicious script tags in their translations. Consider using `force_escape` to explicitly escape a transalted text.
python.django.security.audit.xss.template-translate-as-no-escape
     > python/django/security/audit/xss/template-translate-as-no-escape.html:2
     ╷
    2│   {% translate "Hello world" as the_title %}
    3│   <div>
    4│   <h1>{{ the_title }}</h1>
     ╵
     = Translated strings will not be escaped when rendered in a template. This
       leads to a vulnerability where translators could include malicious script
       tags in their translations. Consider using `force_escape` to explicitly
       escape a transalted text.

python.django.security.audit.xss.template-translate-as-no-escape
     > python/django/security/audit/xss/template-translate-as-no-escape.html:8
     ╷
    8│   {% trans "Hello world" as title %}
    9│   <p>{{ title }}</p>
     ╵
     = Translated strings will not be escaped when rendered in a template. This
       leads to a vulnerability where translators could include malicious script
       tags in their translations. Consider using `force_escape` to explicitly
       escape a transalted text.

